### PR TITLE
complete task 5

### PR DESCRIPTION
**Pull request by Calor94**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -50,6 +50,30 @@ describe("/api/topics", () => {
...
+        expect(body.articles.length).toBe(12);
+        body.articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("number");
```
Comment (HannahBurks) [inline __tests__/app.test.js:71]: I would also add a test to check one of the objects returned and check the comment_count property is correct

File: __tests__/app.test.js [hunk 2]
```diff
@@ -50,6 +50,30 @@ describe("/api/topics", () => {
...
+  test("GET /api/articles responds with status 200 and an array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(12);
+        body.articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
```
Comment (HannahBurks) [inline __tests__/app.test.js:66]: We also want to test that it **doesn't** have a body property. Check what matcher you could use to check this :)

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Just a couple of tests to add - great work though :)

**Comment by cjplatten** [comment]
Nice work!

**Pull request merged**